### DISCUSSION TITLE
Update on PrimeVue Blocks with Tailwind

### DISCUSSION THREAD

**Discussion by M00ker** [primefaces/community#2004] [PrimeVue Blocks] [answered]
Hi,

Please can you give me an update on when you expect to release PrimeVue Blocks with Tailwind support.  

I've purchased PrimeVue Blocks, but I'm having to manually convert the classes from PrimeFlex to Tailwind.

Thanks

**Comment by tugcekucukoglu** [comment, marked as answer]
You don't need to convert them manually. You could use PrimeCLT. for more information: https://v4.primevue.org/primeclt/

**Reply by itd-kogelbauer** [reply]
I get a 404 not found error if i try to open the PrimeCLT link above?

**Reply by almirhodzic** [reply]
You can get the plugin here: https://primevue.org/guides/primeflex/#migration

Additionally, when executing the command “prime pf2tw”, I couldn’t achieve the desired effect, and the conversion or migration from PrimeFlex to Tailwind did not take place. Therefore, it didn’t have the desired effect for me. I’m a bit frustrated.
